google/ads/google_ads/config.py

```diff
@@ -274,33 +279,42 @@ def load_from_env():
 
     if "path_to_private_key_file" in specified_variable_names:
         _logger.warning(
-            "The 'GOOGLE_ADS_PATH_TO_PRIVATE_KEY_FILE' environment "
-            "variable is deprecated. Please use "
-            "'GOOGLE_ADS_JSON_KEY_FILE_PATH' instead."
+            "The 'GOOGLE_ADS_PATH_TO_PRIVATE_KEY_FILE' environment variable "
+            "is deprecated and support will be removed at some point in the "
+            "future. Please use 'GOOGLE_ADS_JSON_KEY_FILE_PATH' instead."
         )
 
     if "delegated_account" in specified_variable_names:
         _logger.warning(
-            "The 'GOOGLE_ADS_DELEGATED_ACCOUNT' environment "
-            "variable is deprecated. Please use "
-            "'GOOGLE_ADS_IMPERSONATED_EMAIL' instead."
+            "The 'GOOGLE_ADS_DELEGATED_ACCOUNT' environment variable "
+            "is deprecated and support will be removed at some point in the "
+            "future. Please use 'GOOGLE_ADS_IMPERSONATED_EMAIL' instead."
         )
 
-    # json_key_file_path is an alternate key that can be used in place of
-    # path_to_private_key_file. It will always override the latter, but the
-    # name will persist for compatibility purposes.
-    if "json_key_file_path" in specified_variable_names:
-        config_data["path_to_private_key_file"] = config_data[
-            "json_key_file_path"
+    # 'json_key_file_path' or GOOGLE_ADS_JSON_KEY_FILE_PATH is the preferred
+    # environment variable to use when setting the location of the json file
+    # used to store service account credentials. It overrides the
+    # 'path_to_private_key_file' variable, which exposes identical
+    # functionality.
```

Comment:
The code below states that the we will use config data of `path_to_private_key_file` if those of `json_key_file_path` are not found.

This paragraph seems to be a bit confusing when reading the code below. Can you reword this or just remove it if not really necessary?

Reply:
Good point, I realize the comments explain the functionality and not what's happening on these lines. I removed them and refactored the code a bit to isolate the logic related to this behavior.